README.md

```diff
@@ -694,3 +694,17 @@ require'nvim-treesitter.configs'.setup {
     --
 }
 ```
+
+#### I experience compilation errors when installing certain parsers
+
+Some parsers do not specify a standard for their compilation and rely on the compiler having a sufficient default.
```

Comment:
It is not about the configs not specifying the config. Even if the parsers would state `c99`. We would be able to set it without the makefile.